db/schema.yml

```diff
@@ -1374,6 +1374,8 @@ hardwares:
 - computer_system_id
 - disk_size_minimum
 - memory_mb_minimum
+- introspected
+- provision_state
```

Comment:
is this concept used elsewhere on other models?
If so, is it possible to reuse that column name?